### DISCUSSION TITLE
blog/post/typescript-extends-is-weird

### DISCUSSION THREAD

**Discussion by giscus** [The-Zagy/zagy-blog#14] [General]
# blog/post/typescript-extends-is-weird

How to think/approach typescript's extends

https://www.zagy.tech/blog/post/typescript-extends-is-weird

**Comment by JoeDDenn** [comment]
Amazing article, well done.
I enjoyed it thoroughly!

**Comment by nagy-nabil** [comment]
Big Philosophy words let's go

**Comment by nagy-nabil** [comment]
Still hate the eco system
But "a extends b" = true if 'a' can fit into 'b' is well said, and i think it's the easiest way to think about it
